host.ts

```diff
@@ -189,6 +191,8 @@ export interface Host {
    * shell's current working directory.
    */
   cwd(): string;
+
+  exit(code: number): void;
```

Comment:
I decided to keep this void, and return in the runtime. This will make the behavior more flexible. However, I'm going to continue throwing a mock exit in `MockConsoleHost`, since it's a sensible behavior and I don't want to refactor the tests.